stm/vernac_classifier.ml

```diff
@@ -161,7 +161,6 @@ let classify_vernac e =
     | VernacExistingClass _ | VernacExistingInstance _
     | VernacRegister _
     | VernacNameSectionHypSet _
-    | VernacDeclareCustomEntry _
```

Comment:
Spurious change?

Reply:
Oh, this is coming from #9421 which was necessary for the CI to pass, but it is now merged on master.

Reply:
I've rebased just in case. By the way I saw a few spurious failures of the test suite, mainly on async.

Reply:
I sometimes see `auto.v`, did you see another one?

Reply:
Some `open_3xxx` I think.